core/src/main/java/com/novoda/noplayer/drm/ModularDrmProvisionRequest.java

```diff
@@ -0,0 +1,54 @@
+package com.novoda.noplayer.drm;
+
+import java.util.Arrays;
+
+public class ModularDrmProvisionRequest {
```

Comment:
Given that these classes are the same as each other I'd probably just have a `ModularDrmRequest` and use it in both places